installation/before-install.md

```diff
@@ -55,5 +55,18 @@ Use `sysctl -p` command or reboot your node for the changes to take effect.
 
 These kernel options were originally taken from the presentation [How Netflix Tunes EC2 Instances for Performance](https://www.slideshare.net/brendangregg/how-netflix-tunes-ec2-instances-for-performance) by [Brendan Gregg](http://www.brendangregg.com/), Senior Performance Architect at AWS re:Invent 2017.
 
-If this article is incorrect or outdated, or omits critical information, please [let us know](https://github.com/fluent/fluentd-docs-gitbook/issues?state=open). [Fluentd](http://www.fluentd.org/) is an open-source project under [Cloud Native Computing Foundation \(CNCF\)](https://cncf.io/). All components are available under the Apache 2 License.
+## Use sticky bit symlink/hardlink protection
+
+Fluentd sometimes uses predictable paths for dumping, writing files, and so on.
+For symlink attack protection, check the following parameters are set up as `1`:
+
+```text
+fs.protected_hardlinks = 1
+fs.protected_symlinks = 1
+```
 
+This settings are almost enough for time-of-check to time-of-use (TOCTOU, TOCTTOU or TOC/TOU) which is a class of software bugs.
+
```

Comment:
It may be better to explain who should customize these parameters.
In the recent version of Linux distribution, it seems already enabled.
(At least Debian 10 (buster) or later and Ubuntu 20.04 or later, these configuration is enabled)

Reply:
Addition, it may be useful if the link to the reference to these parameters.

https://www.kernel.org/doc/Documentation/sysctl/fs.txt

Reply:
> (At least Debian 10 (buster) or later and Ubuntu 20.04 or later, these configuration is enabled)

I'm still stay in Ubuntu 18.04. It is also enabled these parameters.

Reply:
I added a notice for sticky bit protections.

And I noticed that CentOS 6 does not support these parameters.... Oh my gosh.... :confounded: